app/controllers/publishers/vacancies/job_applications/notes_controller.rb

```diff
@@ -1,24 +1,19 @@
 class Publishers::Vacancies::JobApplications::NotesController < Publishers::Vacancies::JobApplications::BaseController
-  def index
```

Comment:
we won't have a notes index page anymore, they are shown on the job application show page